Viewer/src/managers/sceneManager.ts

```diff
@@ -59,6 +59,15 @@ export class SceneManager {
      */
     onVRConfiguredObservable: Observable<IPostConfigurationCallback<VRExperienceHelper, IVRConfiguration>>;
 
+    /**
+     * Will notify when VR mode is entered.
+     */
+    onEnteringVRObservable: Observable<any>;

```

Comment:
the obeservables are not needed here. Since the sceneManager has a (nullable) reference to the observers manager, use it. Dont add another observer which is the same. Notice that the observers that are added to the scene manager are all either configuration or very scene related.

Reply:
I've made the change.